geo-types/src/coordinate.rs

```diff
@@ -9,7 +9,7 @@ use approx::{AbsDiffEq, RelativeEq, UlpsEq};
 /// as an envelope, a precision model, and spatial reference system
 /// information), a `Coordinate` only contains ordinate values and accessor
 /// methods.
-#[derive(Eq, PartialEq, Clone, Copy, Debug, Hash)]
+#[derive(Eq, PartialEq, PartialOrd, Clone, Copy, Debug, Hash)]
```

Comment:
where does `PartialOrd` get used?

Reply:
It is used in fn `lexicographically_least_index` in utils.rs;  that is used for `Winding` and `graham_hull`.

Reply:
Thinking about it,  it does makes things look a bit more magical.  Removed it in favor of describing the order (x then y) explicitly in the utility function.